AssemblerLib/SerialExecutor.h

```diff
@@ -62,6 +62,24 @@ struct SerialExecutor
             f(i, *c[i], std::forward<Args_>(args)...);
     }
 
+    /// Executes the given \c method of the given \c object for each element
+    /// from the input \c container.
+    ///
+    /// This method is very similar to executeDereferenced().
+    ///
+    /// \see executeDereferenced()
+    template <typename Container, typename Object, typename... MethodArgs, typename... Args>
+    static void
+    executeMemberDereferenced(
```

Comment:
removed `execute()`. `executeDereferenced()` is still needed for, e.g., for lambdas or plain functions.